src/core/Akka/Configuration/Pigeon.conf

```diff
@@ -269,6 +269,15 @@ akka {
       }
     }
 
+    channel-scheduler {
```

Comment:
How do `work-max` et al relate to other items such as `dispatcher.throughput` for individual actor mailboxes?

Reply:
It is not related, says only that a worker is processing max count of work items (normal priority) before it loops
The loop forces the worker to check high-priority items

Reply:
In my field test there was only the main worker running. 
This means that only if normal-priority work with `max-work`  takes longer 
as `work-internal / work-step` (250ms) then a coworker gets started and executes the high priority work.